docs/pages/cspell.json

```diff
@@ -0,0 +1,613 @@
+// cSpell Settings
```

Comment:
Was this added intentionally? I'd recommend a separate PR if you want to enable spellchecking.

Reply:
No will remove, thanks!